MediaFileBundle/Manager/UploadedMediaManager.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace OpenOrchestra\MediaFileBundle\Manager;
+
+use Knp\Bundle\GaufretteBundle\FilesystemMap;
+
+/**
+ * Class UploadedMediaManager
+ */
+class UploadedMediaManager
```

Comment:
you could have moved this class in the namespace `MediaFile\Manager`.

This way the bundle would be only the link between the code and the framework